src/client/graphics/layers/OptionsMenu.ts

```
@@ -5,6 +5,9 @@ import { EventBus } from "../../../core/EventBus";
 import { GameType } from "../../../core/game/Game";
 import { GameUpdateType } from "../../../core/game/GameUpdates";
 import { GameView } from "../../../core/game/GameView";
+import { UserSettings } from "../../../core/game/UserSettings";
+import { AlternateViewEvent, RefreshGraphicsEvent } from "../../InputHandler";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'AlternateViewEvent'. Consider to delete it or rename it.

'AlternateViewEvent' is defined here:


(lint/suspicious/noRedeclare)